### ISSUE TITLE
TypeError: this.$initialize is not a function with [ObjectLiteral] inheritance

### ISSUE THREAD

**Issue by geoffreymcgill** [bridgedotnet/Bridge#1819]
Reported by @ProductiveRage on the [forums](http://forums.bridge.net/forum/bridge-net-pro/bugs/2685-open-1819-breaking-change-with-objectliteral-class-inheriting-from-non-objectliteral-class).

### Expected

No JavaScript error

### Actual

JavaScript error

### Steps To Reproduce

http://deck.net/6ec7b41baa416bb4381d7aae683a82b7

```csharp
public class Program
{
    public static void Main()
    {
        var x = new Attributes { Name = "test " };
    }
}

[ObjectLiteral]
public class Attributes : AttributeBase
{
    public string Name { get; set; }
}

public class AttributeBase { }
```

**Comment by LeonidVeriga** [comment]
Related to #1529 and #1758

**Comment by LeonidVeriga** [comment]
Fix details https://github.com/bridgedotnet/Bridge/pull/1824#issue-176764255:
- An `interface` marked with `[ObjectLiteral]` should not contain members;
- `[ObjectLiteral]` with `ObjectCreateMode.Plain` (a default mode):  
  - cannot be inherited from a class with `ObjectCreateMode.Constructor` mode
  - the class definition gets rendered ((`Bridge.define("")`)) only if there are methods (construtors are always ignored for `Plain` mode)
- `[ObjectLiteral]` with `ObjectCreateMode.Constructor`:  
  - can only be inherited from another `ObjectCreateMode.Constructor` class (or from `System.Object`)

**Comment by geoffreymcgill** [comment]
@ProductiveRage - This issue has been fixed and will be included in Bridge 15.0.1 which will be released as a NuGet package. 

The original Deck now compiles properly. 

Thanks for reporting the issue.
